react/react1/week1/todo-tracker/src/components/ToDoRows.jsx

```diff
@@ -0,0 +1,10 @@
+function todoRows(props) {
```

Comment:
Smaller detail, but I suggest you rename it to something like ToDoRow (singular name).
It can be important to use names that make sense, and since this component contains information about a single todo item, you should use singular.